cephfs/admin/fs_quiesce.go

```diff
@@ -0,0 +1,131 @@
+//go:build ceph_preview
+
+package admin
+
+import "fmt"
+
+// FixedPointFloat is a custom type that implements the MarshalJSON interface.
+// This is used to format float64 values to two decimal places.
+// By default these get converted to integers in the JSON output and
+// fail the command.
+type FixedPointFloat float64
+
+// MarshalJSON provides a custom implementation for the JSON marshalling
+// of FixedPointFloat. It formats the float to two decimal places.
+func (fpf FixedPointFloat) MarshalJSON() ([]byte, error) {
+	return []byte(fmt.Sprintf("%.2f", float64(fpf))), nil
+}
+
+// SubVolumeQuiesceFields is the internal type used to create JSON for ceph.
```

Comment:
if this is an internal type, don't export it and call it `subVolumeQuiesceFields` instead.